File: python/rna-transcription/rna_transcription.py
def to_rna(dna_strand):
    dna_to_rna = {'G':'C','C':'G','T':'A','A':'U'} 
    rna = ""
    for c in dna_strand:
        try:
            rna += dna_to_rna[c] 
        except:
            raise ValueError("Given DNS strand is not valid!") 
    return rna
        

File: go/rna-transcription/rna_transcription.go
package strand

import "strings"

func rnaMap(input rune) rune {
	switch input {
	case 'G':
		return 'C'
	case 'C':
		return 'G'
	case 'T':
		return 'A'
	case 'A':
		return 'U'
	default:
		return 'Z'
	}
}

// ToRNA - converting dna string to rna one
func ToRNA(dna string) string {
	return strings.Map(rnaMap, dna)
}
